src/main/java/org/spongepowered/mod/interfaces/IMixinEntity.java

```diff
@@ -35,4 +36,9 @@
     Entity getTeleportVehicle();
 
     void setTeleportVehicle(Entity entity);
+
+    void readFromNbt(NBTTagCompound compound);
```

Comment:
This is not needed. Use the supermixins branch in Mixin then use SoftOverride annotation instead.